src/components/grid-view/GridViewSlider.js

```diff
@@ -0,0 +1,62 @@
+// @flow
```

Comment:
Noticed that there isn't a `Slider` component in ui-elements.  Would it be useful to make a generic `Slider` component and then have `GridViewSlider` consume `Slider`?

Reply:
That's always a good thing to do, you might need to generalize it when you do so. You can always do a follow up PR to split it out